### DISCUSSION TITLE
Is anyone using jruby?

### DISCUSSION THREAD

**Discussion by mnovelo** [rails-on-services/apartment#300] [Polls]
This repository has lingering support for jruby but only with Rails 7...

I do see that a major jruby update is coming https://www.infoworld.com/article/3542893/jruby-10-due-to-arrive-in-early-2025.html

I plan to drop support for jruby once Rails 8 is released, if not sooner.

As such, I'm starting this discussion to hear whether jruby support is

1. not needed
2. needed only for a legacy application that will not be upgrading to Rails 7.1+ any time soon
    - as such, we take responsibility for maintaining a fork of this gem until we can update
3. needed for my active application, and I'm only waiting for jruby 10 to be released to upgrade to Rails 7.1+
    - I also plan to actively help ros-apartment support jruby 10

**Comment by DZverev** [comment]
@mnovelo we need jruby support...

**Reply by mnovelo** [reply]
@DZverev until jruby starts targeting version of Ruby [within these parameters](https://github.com/rails-on-services/apartment/discussions/312#discussioncomment-11836387), we're dropping ongoing support for it. Hopefully jruby 10 drops soon and is easy to add support for it
